worknenjoy/gitpay
# Issue#802- Solving unwanted refresh when the explore button in top bar is clicked

## Description

> Refers to reverted PR #805 

## Changes

- Refactored task-list.js, task-explorer.js and fixing error console in organization-card.js

## Issue

> Closes #802 

## Impacted Area

> Task list
> Task explorer
> Organization list
> Project list
> Organization card
> Top bar

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Import Task
- Click the explore top bar button and see that will not trigger page refresh and will open correctly
- Navigate between project list, organization list to see if works as expected
- Verify if tasks are been filtered correctly

## Before
> Screenshot from the state before

## After
> Screenshot from the state after your Pull Request



https://user-images.githubusercontent.com/21283066/115133242-13943180-9fdd-11eb-9b8e-7df3763186b9.mov
